modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/AbstractMvPartitionStorageTest.java

```diff
@@ -1323,6 +1325,35 @@ void committedMethodCallDoesNotInterfereWithIteratingOverScanCursor(ScanTimestam
         }
     }
 
+    @Test
+    void groupConfigurationOnEmptyStorageIsNull() {
+        assertThat(storage.committedGroupConfiguration(), is(nullValue()));
+    }
+
+    @Test
+    void groupConfigurationIsUpdated() {
+        GroupConfiguration configToSave = new GroupConfiguration(
+                List.of("peer1", "peer2"),
+                List.of("learner1", "learner2"),
+                List.of("old-peer1", "old-peer2"),
+                List.of("old-learner1", "old-learner2")
+        );
+
+        storage.runConsistently(() -> {
+            storage.committedGroupConfiguration(configToSave);
+
+            return null;
+        });
+
+        GroupConfiguration returnedConfig = storage.committedGroupConfiguration();
+
+        assertThat(returnedConfig, is(notNullValue()));
+        assertThat(returnedConfig.peers(), is(List.of("peer1", "peer2")));
```

Comment:
`GroupConfiguration` has `equals` implemented, so you can compare the objects directly